### ISSUE TITLE
GUI.pyw - can't abort started wmlscope

### ISSUE THREAD

**Issue by sevu** [wesnoth/wesnoth#3485]
![abort](https://user-images.githubusercontent.com/21158813/44373531-fc2ddf00-a4e9-11e8-88c5-72e1435535f8.png)

The »X« button does not react.

**Comment by CelticMinstrel** [comment]
Not just wmlscope, this applies to all the tools when run from GUI.pyw.

**Comment by Elvish-Hunter** [comment]
The close button doesn't work because I disabled it in the code. My initial idea was to remove it altogether, but this can't be done in Tkinter. If I didn't disable it, people would've closed the popup and be under the false impression of having stopped the script, which doesn't happen. So, this behaviour is intended.

As for stopping wmlscope (or any other tool), back then I had no idea how to do it, but now I found out that I can send a signal to a subprocess to stop it. This means that in future I'll enable the closing button, as well as implementing a "Stop script" button, but right now this doesn't have a high priority, given that it requires to properly implement #3352.
